OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -253,6 +257,9 @@ public void initializeIAPObserver() {
         iapViewModel.getPurchaseFlowComplete().observe(this, isPurchaseCompleted -> {
             if (isPurchaseCompleted) {
                 fullScreenLoader.dismiss();
+                iapAnalytics.trackIAPEvent(Analytics.Events.IAP_UNLOCK_UPGRADED_CONTENT_TIME, "", "");
+                iapAnalytics.trackIAPEvent(Analytics.Events.IAP_COURSE_UPGRADE_SUCCESS, "", "");
```

Comment:
The `elapsed_time` shows the device time against the `Payments: Time to Unlock Upgraded Content` event when the user but the course from the course component screen.